cmake/TBBBuildInstallScript.cmake

```diff
@@ -0,0 +1,235 @@
+      -DTBB_COMPILER:STRING=clang                                                  \
+      -P cmake/TBBBuildInstallScript.cmake
+"
+)
+
+get_filename_component(TBB_SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/../ ABSOLUTE)
+
+if(NOT MAKE_ARGS)
+  set(MAKE_ARGS "")
+endif()
+if(TBB_COMPILER)
+  list(APPEND MAKE_ARGS "compiler=${TBB_COMPILER}")
+endif()
+if(NOT TBB_BUILD_DIR)
+  set(TBB_BUILD_DIR "${TBB_SOURCE_DIR}/unix_cmake_bld")
+endif()
+list(APPEND MAKE_ARGS "tbb_build_dir=${TBB_BUILD_DIR}")
+if(NOT TBB_BUILD_PREFIX)
+  set(TBB_BUILD_PREFIX "BUILDPREFIX")
+endif()
+list(APPEND MAKE_ARGS "tbb_build_prefix=${TBB_BUILD_PREFIX}")
+
+if(SYSTEM_NAME)
+    set(tbb_system_name ${SYSTEM_NAME})
+else()
+  set(tbb_system_name ${CMAKE_SYSTEM_NAME})
+endif()
+include(${CMAKE_CURRENT_LIST_DIR}/TBBPlatformDefaults.cmake)
+
+if(NOT TBB_RELATIVE_CMAKE_CONFIG_PATH)
+  set(TBB_RELATIVE_CMAKE_CONFIG_PATH "${TBB_SHARED_LIB_DIR}/cmake/TBB" )
+endif()
+
+# "CMAKE_SYSTEM_NAME" not defined by default in scripts executed with -P, use CMAKE_HOST_SYSTEM_NAME
+set(CMAKE_SYSTEM_NAME ${CMAKE_HOST_SYSTEM_NAME})
+include(${CMAKE_CURRENT_LIST_DIR}/TBBBuild.cmake)
+message(STATUS "Building from source directory :${TBB_SOURCE_DIR}:")
+message(STATUS "Building in temporary location: ${TBB_BUILD_DIR} under build prefix ${TBB_BUILD_PREFIX}")
+
+if( NOT DEFINED TBB_BUILD_SHARED OR TBB_BUILD_SHARED )
+  message(STATUS "Building shared libs with MAKE_ARGS:${MAKE_ARGS}:")
+  tbb_build(TBB_ROOT ${TBB_SOURCE_DIR}
+            MAKE_ARGS ${MAKE_ARGS}
+            CONFIG_DIR tbb_config_dir
+            TBB_CMAKE_PACKAGE_CONFIG_DIR ${TBB_BUILD_DIR}/${TBB_RELATIVE_CMAKE_CONFIG_PATH}
+          )
+endif()
+
+if( TBB_BUILD_STATIC )
+  list(APPEND MAKE_ARGS "extra_inc=big_iron.inc") #<- big_iron.inc is included to force build of static libs
+  message(STATUS "Building static libs with MAKE_ARGS:${MAKE_ARGS}:")
+  tbb_build(TBB_ROOT ${TBB_SOURCE_DIR}
+            MAKE_ARGS ${MAKE_ARGS}
+            CONFIG_FOR_INSTALL
+            CONFIG_DIR tbb_config_dir
+            TBB_CMAKE_PACKAGE_CONFIG_DIR ${TBB_BUILD_DIR}/${TBB_RELATIVE_CMAKE_CONFIG_PATH}
+          )
+endif()
+message(STATUS "Configured temporary cmake config files at:${tbb_build_CONFIG_DIR}:") # set from inside tbb_build
+
+##------------------------------------- Install in standard unix directory organization --------------------------------
+file(MAKE_DIRECTORY ${CMAKE_INSTALL_PREFIX})
+file(MAKE_DIRECTORY ${CMAKE_INSTALL_PREFIX}/${TBB_SHARED_LIB_DIR})
+file(MAKE_DIRECTORY ${CMAKE_INSTALL_PREFIX}/${TBB_RELATIVE_CMAKE_CONFIG_PATH})
+
+## First copy the include directory
+file(COPY ${TBB_SOURCE_DIR}/include
+        DESTINATION ${CMAKE_INSTALL_PREFIX}
+        FILE_PERMISSIONS OWNER_READ GROUP_READ WORLD_READ OWNER_WRITE
+        DIRECTORY_PERMISSIONS OWNER_READ GROUP_READ WORLD_READ OWNER_EXECUTE GROUP_EXECUTE WORLD_EXECUTE OWNER_WRITE
+        )
+
+if (CMAKE_SIZEOF_VOID_P EQUAL 8)
```

Comment:
Is it a real use case to install x64/x86 binaries into one `CMAKE_INSTALL_PREFIX` but in different subdirectories?

Reply:
@AlexVeprev I don't understand the question in the context pointed to.  Sorry.